lib/content-services/breadcrumb/breadcrumb.component.html

```diff
@@ -1,29 +1,53 @@
 <div *ngIf="folderNode" data-automation-id="breadcrumb" class="adf-breadcrumb-container">
-    <li *ngFor="let item of route; let last = last"
-        [class.active]="last"
-        [ngSwitch]="last"
-        title="{{ item.name | translate }}"
-        class="adf-breadcrumb-item">
-
-        <a *ngSwitchDefault href="#" [attr.data-automation-id]="'breadcrumb_' + item.name"
-           class="adf-breadcrumb-item-anchor"
-           (click)="onRoutePathClick(item, $event)">
-            {{ item.name | translate }}
-        </a>
-
-        <div *ngSwitchCase="true" class="adf-breadcrumb-item-current">
-            {{ item.name | translate }}
-        </div>
-
-        <mat-icon class="adf-breadcrumb-item-chevron" *ngIf="!last">
-            chevron_right
-        </mat-icon>
-    </li>
-</div>
-<div *ngIf="!folderNode && hasRoot">
-    <li class="adf-breadcrumb-item">
-        <div class="adf-breadcrumb-item-current">
-            {{ root | translate }}
-        </div>
-    </li>
-</div>
+        <button
+            *ngIf="hasPreviousNodes()"
+            tabindex="0"
+            class="adf-breadcrumb-dropdown-trigger"
+            (click)="open()">
+            <mat-icon [class.isRoot]="!hasPreviousNodes()">folder</mat-icon>
+            <mat-icon [class.isRoot]="!hasPreviousNodes()" class="adf-breadcrumb-dropdown-trigger-arrow">arrow_drop_down</mat-icon>
+        </button>
+    
+        <mat-select
+            #select
+            *ngIf="hasPreviousNodes()"
+            class="adf-breadcrumb-dropdown-path"
+            tabindex="0">
+    
+            <mat-option
+                *ngFor="let node of previousNodes;"
+                (click)="onRoutePathClick(node, $event)"
+                class="adf-breadcrumb-path-option"
+                tabindex="0">
+                {{ node.name }}
+            </mat-option>
+        </mat-select>
+    
+        <li *ngFor="let item of lastNodes; let last = last"
+            [class.active]="last"
+            [ngSwitch]="last"
+            title="{{ item.name | translate }}"
+            class="adf-breadcrumb-item">
+    
+            <a *ngSwitchDefault href="#" [attr.data-automation-id]="'breadcrumb_' + item.name"
+               class="adf-breadcrumb-item-anchor"
+               (click)="onRoutePathClick(item, $event)">
+                {{ item.name | translate }}
+            </a>
+    
+            <div *ngSwitchCase="true" class="adf-breadcrumb-item-current">
+                {{ item.name | translate }}
+            </div>
+    
+            <mat-icon class="adf-breadcrumb-item-chevron" *ngIf="!last">
+                chevron_right
+            </mat-icon>
+        </li>
+    </div>
+    <!-- <div *ngIf="!folderNode && hasRoot">
```

Comment:
can you remove this HTML commented from here? In general please try to not commit commented code anywhere thanks